.dockstore.yml

```diff
@@ -0,0 +1,37 @@
+version: 1.2
+workflows:
+   - subclass: WDL
+     primaryDescriptorPath: /variant-caller/variant-caller-wdl/topmed_freeze3_calling.wdl
+     testParameterFiles:
```

Comment:
don't you want to use the newer https://github.com/DataBiosphere/topmed-workflow-variant-calling?

Reply:
Good spot, thanks!

Reply:
Actually since that's a different repo I'm a little hesistant. Does that one have its own page on Dockstore?